app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -110,5 +110,13 @@
 
   <div class="govuk-grid-column-one-third">
     <%= render partial: '/candidate_interface/shared/need_help', locals: { support_reference: @application_form.support_reference } %>
+
+    <% if FeatureFlag.active?('apply_again') && @previous_application_form.present? %>
+      <h2 class="govuk-heading-s">Previous applications</h2>
+      <%= govuk_link_to 'First application', candidate_interface_application_review_previous_path(@previous_application_form.id) %>
+      <p class="govuk-body-s">
+        <%= "Submitted #{@previous_application_form.submitted_at.to_s(:govuk_date).strip}" %>
+      </p>
```

Comment:
Can we do the work now to make this work for N applications, or is it tricky currently? I think we could enumerate `current_candidate.application_forms.sort_by(&:submitted_at)` to come up with a list, `each` them, and then use the inflector to numeralize the link?

Reply:
Plus to ease testing burden, it could be extracted as a component and unit tested

Reply:
I don't see any reason this can't be done now